sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs

```diff
@@ -16,6 +16,7 @@
 using Azure.Storage.Test.Shared;
 using Moq;
 using NUnit.Framework;
+using static Azure.Storage.Blobs.Tests.ClientBuilderExtensions;
```

Comment:
Why is the import static?

Reply:
This is just what visual studio decided to do when using alt+enter to import the missing stuff. Should we not be doing this?

Reply:
If regular import works instead, we should use regular import for consistency.